common/src/main/java/com/revenuecat/purchases/common/Config.kt

```diff
@@ -4,5 +4,5 @@ object Config {
 
     var debugLogsEnabled = false
 
-    const val frameworkVersion = "4.7.0-SNAPSHOT"
+    const val frameworkVersion = "5.0.0-RC1"
```

Comment:
do we know if gradle will suggest this as an upgrade? probably, right?

Reply:
Probably because it was suggesting the `alpha` ones